From 02763d9a609d902b1efdae071a46a1825bdb8884 Mon Sep 17 00:00:00 2001
From: Bharat Bhushan <Bharat.Bhushan@nxp.com>
Date: Thu, 31 May 2018 17:22:16 +0800
Subject: [PATCH 337/448] virt/vgic: Increase number of DeviceIDs to 17

FSL-MC bus devices uses device-ids from 0x10000 to 0x20000.
So to support MSI interrupts for mc-bus devices we need
vgi-ITS device-id table of size 2^17 to support deviceid
range from 0x10000 to 0x20000.

Signed-off-by: Bharat Bhushan <Bharat.Bhushan@nxp.com>
Integrated-by: Zhao Qiang <qiang.zhao@nxp.com>
---
 virt/kvm/arm/vgic/vgic-its.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/virt/kvm/arm/vgic/vgic-its.c b/virt/kvm/arm/vgic/vgic-its.c
index 5050b58..82b698a 100644
--- a/virt/kvm/arm/vgic/vgic-its.c
+++ b/virt/kvm/arm/vgic/vgic-its.c
@@ -176,7 +176,7 @@ struct its_itte {
 
 #define GIC_LPI_OFFSET 8192
 
-#define VITS_TYPER_DEVBITS 16
+#define VITS_TYPER_DEVBITS 17
 
 /*
  * Finds and returns a collection in the ITS collection table.
-- 
1.7.1

